bottlerocket-settings-models/modeled-types/src/kubernetes.rs

```diff
@@ -353,14 +358,14 @@ mod test_kubernetes_cluster_name {
 
     #[test]
     fn good_cluster_names() {
-        for ok in &["more-chars_here.now", &"a".repeat(63)] {
+        for ok in &["more-chars_here-123", &"a".repeat(100)] {
```

Comment:
`.` no longer being an allowed character is the only notable change in this PR, I think